ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/SubscriptionDetailsView.kt

```diff
@@ -148,6 +148,7 @@ private class SubscriptionInformationProvider : PreviewParameterProvider<Purchas
             expirationDateString = "June 1st, 2024",
             willRenew = true,
             active = true,
+            productId = "basic_monthly",
```

Comment:
I just noticed that this is a `PreviewParameterProvider` and was gonna comment that Emerge doesn't support those. However it seems [they do support it since version 1.3.0](https://docs.emergetools.com/docs/android-snapshots-v1#previewparameter-annotated-parameters) of their Snapshots SDK. We're using 1.2.0. Could you bump? 😄

Reply:
done in [`d20b581` (#2008)](https://github.com/RevenueCat/purchases-android/pull/2008/commits/d20b5811de9f56b36cbc2865ae157585d17515a1)